### PR TITLE
fix(DB/Creature): removed skull visual modification

### PR DESCRIPTION
**Pull request by Knindzagxg**
Removing skull visual modification from previous Dungeons Boss NPC's.

To test:

1. Apply SQL
2. Restart server & delete your cache
3. Go to any dungeon (Classic, TBC or WoTLK) and check if boss have skull or level number.

### CODE REVIEW

**Review by FrancescoBorzi — APPROVED** [review]

File: data/sql/updates/pending_db_world/rev_1606146563846030900.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
...
...3886,3887,4278,4279,4274,3927,4275,1696,1666,1663,1716,1717,10435,10808,11032,10997,10811,10812,10813,10436,10437,10438,10439,10440,8580,8443,5709,5710,5711,5712,5713,5714,5715,5716,5717,5719,5720,5721,5722,6910,6906,6907,6908,7206,7291,4854,7023,7228,2748,9816,10264,10429,10430,10339,103363,3671,3670,3674,3673,5775,3653,3669,3654,7272,8127,7271,7604,7795,7796,7275,7273,7797,7267,18371,20318,18373,20306,17381,18621,17380,18601,17377,18607,17879,20738,17880,20745,17881,20737,17876,20548,17848,20535,17862,20521,18096,20531,17306,18436,17308,18433,17536,18432,17537,18434,24723,25562,24744,25573,24559,25574,24558,25549,24554,25550,24555,25555,24561,25578,24553,25541,24556,25579,24557,25556,24560,25560,24664,24857,18341,20267,18343,20268,18344,20266,22930,18472,20690,18473,20706,23035,18731,20636,18667,20637,18732,20653,18708,20657,16807,20568,16809,20596,16808,20597,20923,20993,17941,19893,17991,19895,17942,19894,17797,20629,17796,20630,17798,20633,20870,21626,20885,21590,20886,21624,20912,21601,21466,21600,21467,21599,17976,21551,17975,21558,17978,21581,17980,21559,17977,21582,19710,21526,19219,21533,19218,21525,19221,21536,19220,21537,17770,20169,18105,20168,17826,20183,17882,20184);
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1606146563846030900.sql:2]:
```suggestion
UPDATE `creature_template` SET `type_flags`=`type_flags`&~ 4 WHERE `entry` IN (4887,4831,6243,4830,4832,4829,16042,8923,8929,8983,9016,9017,9018,9019,9026,9027,9028,9029,9030,9031,9032,9033,9034,9035,9036,9037,9038,9039,9040,9056,9156,9319,9499,9502,9537,9543,9938,16059,16080,644,642,643,1763,647,639,646,645,14354,14327,13280,11490,11492,14322,14326,14321,14323,14325,14324,11501,11489,11487,11488,11496,11486,14506,7361,7079,6235,6229,7800,16387,9196,9237,9236,9736,10596,10584,10220,10268,9568,13282,13596,13601,12258,12236,12225,12203,12201,11520,11518,11519,11517,7355,7357,8567,7356,7358,4424,4428,4420,4422,4421,6168,3983,4543,3974,6487,3975,4542,3976,3977,10506,11622,10433,10432,10508,10503,10901,10502,10504,10507,11261,10505,1853,3914,3865,3886,3887,4278,4279,4274,3927,4275,1696,1666,1663,1716,1717,10435,10808,11032,10997,10811,10812,10813,10436,10437,10438,10439,10440,8580,8443,5709,5710,5711,5712,5713,5714,5715,5716,5717,5719,5720,5721,5722,6910,6906,6907,6908,7206,7291,4854,7023,7228,2748,9816,10264,10429,10430,10339,103363,3671,3670,3674,3673,5775,3653,3669,3654,7272,8127,7271,7604,7795,7796,7275,7273,7797,7267,18371,20318,18373,20306,17381,18621,17380,18601,17377,18607,17879,20738,17880,20745,17881,20737,17876,20548,17848,20535,17862,20521,18096,20531,17306,18436,17308,18433,17536,18432,17537,18434,24723,25562,24744,25573,24559,25574,24558,25549,24554,25550,24555,25555,24561,25578,24553,25541,24556,25579,24557,25556,24560,25560,24664,24857,18341,20267,18343,20268,18344,20266,22930,18472,20690,18473,20706,23035,18731,20636,18667,20637,18732,20653,18708,20657,16807,20568,16809,20596,16808,20597,20923,20993,17941,19893,17991,19895,17942,19894,17797,20629,17796,20630,17798,20633,20870,21626,20885,21590,20886,21624,20912,21601,21466,21600,21467,21599,17976,21551,17975,21558,17978,21581,17980,21559,17977,21582,19710,21526,19219,21533,19218,21525,19221,21536,19220,21537,17770,20169,18105,20168,17826,20183,17882,20184);
```

**Review by LukasVolgger — APPROVED** [review]

**Comment by BarbzYHOOL** [comment]
thank you

**Comment by LukasVolgger** [comment]
Tested and working.

/* Betroffene Zeilen: 301  Gefundene Zeilen: 0  Warnungen: 0  Dauer von 2 Abfragen: 0,062 Sek. */

Are these all bosses? It seems some are missing. Like guid 48706 in RFC?
image

**Comment by BarbzYHOOL** [comment]
the boss list has been given by knindza, he might have missed some :'(((((((

can't we simply verify with a smart sql query with `WHERE flag = flag | 4` ? can u check lukas?

**Comment by LukasVolgger** [comment]
> the boss list has been given by knindza, he might have missed some :'(((((((
> 
> can't we simply verify with a smart sql query with `WHERE flag = flag | 4` ? can u check lukas?

Sure, i'll check this tonight so Knindza can play Shadowlands :P👍🏻

**Comment by Knindzagxg** [comment]
I will sort today after work.

It's weird tho, i used all my list i had from my PR's tho xP

**Comment by Knindzagxg** [comment]
Updated Taragaman the Hungerer (11520) from RFC.
Was he the only one there btw?

Think checking with SQL won't work cause raid bosses will be show as well.

**Comment by BarbzYHOOL** [comment]
> Updated Taragaman the Hungerer (11520) from RFC.
> Was he the only one there btw?
> 
> Think checking with SQL won't work cause raid bosses will be show as well.

sure raid bosses will be shown, but you guys are wotlk specialists so you should easily distinguish what is "right" or "wrong" :P
there shouldn't be more than 100 NPCs approximatively, imo
select them by name, subname and minlevel and should be easy to filter

**Comment by BarbzYHOOL** [comment]
ok, cause i'm extra cool i made the query for you:

```SELECT entry, name, subname, minlevel, maxlevel FROM creature_template WHERE type_flags = type_flags | 4```

You can test ig the first mob `.npc add temp 25` it's a mob that has the skull level and shouldn't even exist imo, lol probably from the vanilla beta

**Comment by Voxstrasza** [comment]
> 
> 
> ok, cause i'm extra cool i made the query for you:
> 
> `SELECT entry, name, subname, minlevel, maxlevel FROM creature_template WHERE type_flags = type_flags | 4`
> 
> You can test ig the first mob `.npc add temp 25` it's a mob that has the skull level and shouldn't even exist imo, lol probably from the vanilla beta

I applied the SQL for the PR, ran this and am sifting through the list - but it's pretty hefty.  I'll post my findings when I'm complete

**Comment by Knindzagxg** [comment]
Raids are not modified, and visual skulls are there as always.

We just had lots of PRs to change Dungeon boss npc and to give them visual modification and now removing xP
So just check dungeons if possible.

**Comment by Voxstrasza** [comment]
> 
> 
> Raids are not modified, and visual skulls are there as always.
> 
> We just had lots of PRs to change Dungeon boss npc and to give them visual modification and now removing xP
> So just check dungeons if possible.

just finished sifting through the DB, all looks good 👍🏻

**Review by Helias — APPROVED** [review]

**Comment by Helias** [comment]
Thanks @Knindzagxg  and thx everyone for testing and reviewing, I think we can put "To Be Merged" here

**Review by Si1ker — APPROVED** [review]

**Pull request merged**